common/log.h

```diff
@@ -65,7 +65,7 @@ extern int isIni;
 #define ASSERT(expr) do {                                                                                               \
         if (!(expr))                                                                                                    \
             ERROR();                                                                                                    \
-        assert(expr);                                                                                                   \
+            assert(0 && expr);                                                                                                   \
```

Comment:
Did you intend to move this into the conditional body?  If so, you forgot to enclose the body in braces.